src/css/button.scss

```diff
@@ -31,4 +32,12 @@ button, .button {
             transform: none;
         }
     }
+
+    @include media-breakpoint-down('sm') {
+        width: 100%;
+        padding: 20px 14px;
+        overflow: hidden;
+        text-overflow: ellipsis;
```

Comment:
Should we allow these to wrap? /cc @Gluzman these are the buttons; e.g.:

![Narrow button leading to ellipsis instead of wrapping](https://user-images.githubusercontent.com/8245/88543637-424d2080-cfe6-11ea-8708-3eb97829b36f.png)